modules/mod_logging/templates/admin_log_ui.tpl

```diff
@@ -0,0 +1,73 @@
+{% extends "admin_log_base.tpl" %}
+
+{% block title %}{_ Log ui events _}{% endblock %}
+
+{% block title_log %}{_ Log user-interface events _}{% endblock %}
+
+{% block active3 %}active{% endblock %}
+
+{% block content_log %}
+
+<h3 class="above-list">
+    {_ Most recent user interface events _}
+</h3>
+<br />
+
+<form id="log_filter" action="" type="GET">
+{% with m.search[{log_ui page=q.page type=q.type user=q.user pagelen=100}] as result %}
+    <table class="table table-compact">
+        <tr>
+            <th>{_ Date _}</th>
+            <th>{_ Severity _}</th>
+            <th>{_ Message _}</th>
+            <th>{_ User _}</th>
+            <th>{_ Stack _}</th>
+        </tr>
+        <tr>
+            <td></td>
+            <td>
+                <select name="type" id="log_severity" class="form-control">
+                    <option {% if q.type == 'debug' %}selected{% endif %} value="debug">Debug</option>
```

Comment:
No translations here?

Reply:
Copied this from the other log screen, also no translations there.
We might want to add some, though I think this was left untranslated so that it matches the texts in the log rows underneath (those come from the database and are left as-is).